src/query/service/src/interpreters/interpreter_unsetting.rs

```diff
@@ -92,9 +93,10 @@ impl Interpreter for UnSettingInterpreter {
             }
         }
         self.ctx.set_affect(QueryAffect::ChangeSettings {
-            keys,
+            keys: keys,
```

Comment:
```suggestion
            keys,
```

Reply:
github seems got a bug on display the diff... 

when I pulled the code in my laptop, it shows `keys` but not `keys: keys`...